profiles/package.mask

```diff
@@ -1,3 +1,64 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-10)
+# The older versions of these packages do not compile with some of the newer
+# dependencies that have been added.
+<dev-haskell/persistent-test-2.0.3.5
+<dev-haskell/persistent-postgresql-2.11.0.1
+<dev-haskell/persistent-sqlit-2.11.0.0
+<dev-haskell/persistent-mysql-2.10.3.1
+<dev-haskell/esqueleto-3.4.0.1
+
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-10)
+# this version of reflex is not currently compatible with ghc 8.10.x, but there
+# [is an pull request][1] open upstream to resolve this.
+# [1]: https://github.com/reflex-frp/reflex/pull/444
+=dev-haskell/reflex-0.8.0.0
+
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-09)
+# Per the description, this is simply a "convenient wrapper" around some other
+# diagrams-* packages. This is currently masked due to being unmaintained, and
+# not pulling in newer versions of the packages it wraps. An upstream issue has
+# been opened [here](https://github.com/diagrams/diagrams/issues/25)
+>=dev-haskell/diagrams-1.4
+>=dev-haskell/ihaskell-diagrams-0.3.2.1
+
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-05)
+# There is a newer version available, and this older version will not work with
+# hashable-1.3 without tweaking the ebuild. Instead, we'll mask for removal.
+=dev-haskell/linear-1.20.9
```

Comment:
Is there a reason why we cannot remove this ebuild instead of masking it? If there are no revdeps requiring this version (which I believe to be true, since in ::haskell we can't bump a package's major version if there are any reverse dependencies requiring the older version) I say `git rm linear-1.20.9.ebuild && repoman commit -m "dev-haskell/linear: drop old"`

Reply:
I thought that there was some minimum amount of time needed before removing an ebuild, [the devmanual](https://devmanual.gentoo.org/ebuild-maintenance/removal/index.html) says 30 days. Is this more of a guideline than a hard and fast rule?

Reply:
1. Removing ebuild is different (and simpler) than removing a package.
2. Overlay does not have to keep as strict rules as `::gentoo` (but it's a good idea to always consider following the rules)

In this case you are removing the ebuild version, not a full package.

Reply:
I just took this commit out entirely, turns out there's no issue keeping the old one (i think)